docs/interpreter/neo4j.md

```diff
@@ -44,6 +44,11 @@ The Neo4j Interpreter supports all Neo4j versions since v3 via the official [Neo
     <td>bolt://localhost:7687</td>
     <td>The Neo4j's BOLT url.</td>
   </tr>
+  <tr>
+    <td>neo4j.database</td>
+    <td></td>
+    <td>The neo4j target database, if empty use the dafault db.</td>
+  </tr>
```

Comment:
`interpreter-setting.json` also needs to be updated

Reply:
done